pkgs/by-name/fa/fastapi-dls/package.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  python3,
+  fetchFromGitLab,
+  gitUpdater,
+  makeWrapper,
+}:
+
+let
+
+  pythonEnv = python3.withPackages (
+    packages:
+    with packages;
+    [
+      fastapi
+      uvicorn
+      python-jose
+      pycryptodome
+      python-dateutil
+      sqlalchemy
+      markdown
+      python-dotenv
+      python-multipart
+    ]
+    ++ uvicorn.optional-dependencies.standard
+  );
+
+  version = "1.4.1";
+
+in
+python3.pkgs.buildPythonApplication {
+  pname = "fastapi-dls";
+  inherit version;
+  pyproject = false;
+
+  src = fetchFromGitLab {
+    domain = "git.collinwebdesigns.de";
+    owner = "oscar.krause";
+    repo = "fastapi-dls";
+    rev = "refs/tags/${version}";
```

```suggestion
    tag = version;
```
This has been recently introduced and is lighter.

---

Actually I just rebased to HEAD and it is now not building. `fetchFromGitLab` doesn't support it 😂 Only `fetchgit{,hub}` support this in cb9f9a1e5a51a87d59b373db7016cee1608debd5

---

Oh sure, I didn't pay attention.

---

It now does ^^